distribution/docs/src/main/jdocs/content/_transformers/atom-query-response-transformer-contents.adoc

```diff
@@ -0,0 +1,175 @@
+:title: Atom Query Response Transformer
+:type: transformer
+:status: published
+:link: _atom_query_response_transformer
+:summary: Transforms a query response into an http://tools.ietf.org/html/rfc4287[Atom 1.0] feed.
+:input:
+:metacard:
+:queryresponse: x
```

Comment:
Just making three separate types out the transformers because it's impossible to be more than one type of transformer.